sdks/java/harness/src/main/java/org/apache/beam/fn/harness/logging/BeamFnLoggingClient.java

```diff
@@ -97,22 +93,20 @@
   private final Collection<Logger> configuredLoggers;
   private final Endpoints.ApiServiceDescriptor apiServiceDescriptor;
   private final ManagedChannel channel;
-  private final StreamObserver<BeamFnApi.LogEntry.List> outboundObserver;
+  private final ClientCallStreamObserver<BeamFnApi.LogEntry.List> outboundObserver;
```

Comment:
Would CallStreamObserver be sufficient (here and elsewhere)?

Reply:
Yes